package.json

```diff
@@ -40,7 +40,7 @@
     "bpf-sdk:install": "npm run clean:fixtures; bin/bpf-sdk-install.sh .",
     "bpf-sdk:remove-symlinks": "find bpf-sdk -type l -print -exec cp {} {}.tmp \\; -exec mv {}.tmp {} \\;",
     "build": "cross-env NODE_ENV=production rollup -c",
-    "build:fixtures": "./test/fixtures/noop-c/build.sh; ./test/fixtures/noop-rust/build.sh",
+    "build:fixtures": "echo blah",
```

Comment:
I'm guessing you didn't mean to push this change?

Reply:
I'm glad I went with blah this time! 😆

Reply:
🤬 🧼

Reply:
🤐 fixed!